sysutils/apl_select/Portfile

```diff
@@ -0,0 +1,28 @@
+PortSystem 1.0
```

Comment:
This file should start with the usual `#` prefixed modeline. It's also got some indentation problems.

Reply:
Done.